source/help/messaging/mentioning-teammates.rst

```diff
@@ -61,4 +75,18 @@ You can add a list of customized words to get mention notifications for by typin
 Recent Mentions
 ---------------
 
-Click **@** next to the search box to query for your most recent @mentions and words that trigger mentions. Click **Jump** next to a search result in the right-hand sidebar to jump the center pane to the channel and location of the message with the mention.
+Click **@** next to the search box to query for your most recent @mentions and words that trigger mentions. Click **Jump** next to a search result in the right-hand sidebar to jump the center pane to the channel and location of the message with the mention. *Note: this does not include LDAP group mentions*
+
+Confirmation dialogue warnings
+---------------
+
+Any mention that will trigger notifications for over five users, a confirmation dialog will appear requiring confirmation from the user posting the mention before sending notifications to those users. 
+*Dialog will only appear when the following setting is enabled: TeamSettings.EnableConfirmNotificationsToChannel*
+*This warning is not displayed for group mentions on mobile*
```

Comment:
```suggestion
*This warning is not displayed for group mentions on mobile*
```
Do we need to add this, if the feature isn't supported on mobile yet?

Reply:
@justinegeffen the feature is supported on mobile, but this part of it is not.

Reply:
Gotcha. :)

Reply:
I've added a suggestion above; if you're happy with it then we can remove this line as it's added to the previous one.

Reply:
agreed. will remove this line